src/software/sensor_fusion/sensor_fusion.h

```diff
@@ -149,4 +152,6 @@ class SensorFusion
 
     // The timestamp, in seconds, of the most recently received vision packet
     double last_t_capture;
+
+    static constexpr int NUM_DROPPED_DETECTIONS_BEFORE_BALL_NOT_IN_DRIBBLER = 3;
```

Comment:
should this be part of sensor fusion config?

Reply:
added